recipes/aha/meta.yaml

```
@@ -0,0 +1,37 @@
+{% set version = "0.5.0" %}
+{% set major,minor,patch = version.split('.') %}
+
+package:
+  name: aha
+  version: {{ version }}
+
+source:
+  url: https://github.com/theZiz/aha/archive/0.5.tar.gz
```

```suggestion
  url: https://github.com/theZiz/aha/archive/{{ version }}.tar.gz
```

This will let the bot update the recipe.